### ISSUE TITLE
⌨ Accessibility Issue: missing alt attribute on IAP home page images

### ISSUE THREAD

**Issue by Elisa0122** [GCTC-NTGC/gc-digital-talent#6246]
## ❌ Current behaviour (issue)
When voice-over is used, it read the image without the description.

**WCAG 2.1 Criteria:** 1.1.1 non-text Content: All [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG22/#dfn-text-alternative) that serves the equivalent purpose.

### How to reproduce

1. Go to https://uat-talentcloud.tbs-sct.gc.ca/en/indigenous-it-apprentice
2. use apple Voice Over on chrome
3. Start reading by using VO + A

## ✔ Expected behaviour (fix)

1. The image must be read and described.
2. The alt attributs of the images are: 

1.<img width="1686" alt="Screenshot 2023-04-11 at 2 50 02 PM" src="https://user-images.githubusercontent.com/83783101/231260553-b7468117-cfb6-4a32-9eec-f2693445a3ca.png">
1.alt text: Group of Indigenous people standing with a painted hand drum. 

2.<img width="574" alt="Screenshot 2023-04-11 at 2 51 56 PM" src="https://user-images.githubusercontent.com/83783101/231260906-fb9defbc-08cf-4776-886a-60978f45768a.png">
2.alt text:  Indigenous woman wearing a jean jacket which contains several different pins.

2b. alt text: Two feathers tied together. 

3.<img width="669" alt="Screenshot 2023-04-11 at 2 52 16 PM" src="https://user-images.githubusercontent.com/83783101/231260962-a61d6ed5-22a0-4aa7-b767-7fa59644a08e.png">
3.alt text:Indigenous man working at a computer. 

3b. alt text: Métis style gloves with floral beading.

4.<img width="476" alt="Screenshot 2023-04-11 at 2 52 37 PM" src="https://user-images.githubusercontent.com/83783101/231261052-a3ab92ac-bf52-4c69-b280-018c84bbb8a2.png">
4.alt text: Indigenous woman smiling, wearing a brown sweater and glasses.

5.<img width="496" alt="Screenshot 2023-04-11 at 2 52 53 PM" src="https://user-images.githubusercontent.com/83783101/231261116-e3493fe0-93ba-4f26-8037-ae186ed075b5.png">
5.alt text: Ulu, an Inuit tool used by Inuit women.

6.<img width="1535" alt="Screenshot 2023-04-11 at 2 53 14 PM" src="https://user-images.githubusercontent.com/83783101/231261201-6909d32f-2b27-4bb0-a8d0-60d9b4a06efd.png">
6.alt text (background image): Talking stick.

7.<img width="129" alt="Screenshot 2023-04-11 at 2 53 41 PM" src="https://user-images.githubusercontent.com/83783101/231261308-fa1cb140-ab40-4280-8c98-8ce4c91977cb.png">
7.alt text: open quote mark

8.<img width="158" alt="Screenshot 2023-04-11 at 2 54 06 PM" src="https://user-images.githubusercontent.com/83783101/231261442-168ead94-f5ec-4b61-8d71-568cd2d68527.png">
8.alt text: closed quote mark

9.<img width="1776" alt="Screenshot 2023-04-11 at 2 55 17 PM" src="https://user-images.githubusercontent.com/83783101/231261712-26d17245-6c1f-4acb-ab9b-6967136c5c47.png">
9.alt text: Male Traditional dancer in regalia.

9b (background image): Traditional Métis sash 

10.<img width="665" alt="Screenshot 2023-04-11 at 3 00 40 PM" src="https://user-images.githubusercontent.com/83783101/231262998-7761240c-fd8e-4668-bf69-dc453640ee81.png">
10.alt text:  Indigenous woman wearing a red shirt working on a laptop. 

### Acceptance Criteria

A set of assumptions which, when tested, verify that the accessibility requirement was met.

- [ ] Apple Voice Over screen reader can programmatically determine that there is an image.
- [ ]  Apple Voice Over screen reader user hears  the description text of each image.

## Additional information (optional)

1. How does it impact users?  
The image is an important part of the content. It is important to include descriptive alt text for images to ensure that all users, regardless of their abilities or technical limitations, can access and understand the content on the web.

2. Related issues, suggestions how to fix. 
The image must be read and described with the alt attribute corresponding to each image

Indigenous languages to come at a later date, so out of scope for this issue

Convert DIV to IMG elements.  May need to work a bit to preserve layout.

**Comment by esizer** [comment]
@BrookeCameron and @MCommonda Do you have some suggestions for alt test for these images?

**Comment by BrookeCameron** [comment]
Updated alt text by @MCommonda

**Comment by BrookeCameron** [comment]
@gobyrne Should we get this copy translated / reviewed in FR by Nathan?

**Comment by RM-1978** [comment]
Hi, 
I really like the Alt texts above!  My only question is  @esizer, are we able to separate out the alt texts for two  objects within one image? (_e.g., 2.alt text: Indigenous woman wearing a jean jacket which contains several different pins. 2b. alt text: Two feathers tied together._) If we might not have this flex, then I suggest that we include both descriptive texts sequentially inside the same Alt text string.

**Comment by MCommonda** [comment]
Here is the French translation for this Alt text:
[IAP Homepage - Alt Text-FR.docx](https://github.com/GCTC-NTGC/gc-digital-talent/files/12598125/IAP.Homepage.-.Alt.Text-FR.docx)
